libs/plutus-preprocessor/src/Debug.hs

```diff
@@ -7,4 +7,4 @@ import Cardano.Ledger.Plutus.Evaluate (debugPlutus)
 import System.Environment (getArgs)
 
 main :: IO ()
-main = print . debugPlutus @StandardCrypto . head =<< getArgs
+main = mapM_ (print . debugPlutus @StandardCrypto) =<< getArgs
```

Comment:
As discussed during the meeting: this should be moved from here in a separate PR and `opt-parse-applicative` should be used for more granularity.

Reply:
I believe this already has an issue (#3920) so I'll resolve this conversation